frontend/.pa11yci-desktop.json

```diff
@@ -2,7 +2,7 @@
   "defaults": {
     "timeout": 240000,
     "runners": ["axe"],
-    "ignore": ["color-contrast"],
+    "ignore": ["color-contrast", "document-title"],
```

Comment:
didn't want to do this, but couldn't find an easy way around it. When the unauthenticated version of the pages load, the metadata does not, even though as far as I could tell the Next internals pick it up correctly.

Seems likely to be related to the next bug mentioned here https://github.com/vercel/next.js/issues/77512

Reply:
Yeah, good idea https://github.com/HHS/simpler-grants-gov/issues/4495